pom.xml

```diff
@@ -200,4 +181,40 @@
             </plugin>
         </plugins>
     </build>
+
+    <profiles>
+        <profile>
+            <id>Release</id>
+            <activation>
+                <activeByDefault>false</activeByDefault>
```

Comment:
You don't need this, profiles are not active by default.

Reply:
I know, I wanted to be explicit

Reply:
IMHO, useless noise.